apps/meteor/ee/app/livechat-enterprise/server/lib/Helper.ts

```diff
@@ -111,34 +138,34 @@ const dispatchWaitingQueueStatus = async (department) => {
 
 	const queueInfo = await getQueueInfo(department);
 	queue.forEach((inquiry) => {
-		dispatchInquiryPosition(inquiry, queueInfo);
+		void dispatchInquiryPosition(inquiry, queueInfo);
```

Comment:
Considering this is an async function, WDYT about using Promise.all() to make this entire function sync?

Reply:
Makes sense :+1: